fuel-tests/tests/tx/utxo_validation.rs

```diff
@@ -100,9 +100,10 @@ async fn submit_utxo_verified_tx_with_min_gas_price() {
         }
 
         // Once https://github.com/FuelLabs/fuel-core/issues/50 is resolved this should rely on the Submitted Status rather than Success
+
         assert!(matches!(
             transaction_result,
-            TransactionStatus::Success { .. }
```

Comment:
The test should be positive, you need to increase the gas limit

Reply:
fixed!